pkg/controller/datadogagent/datadogagent_controller.go

```diff
@@ -103,7 +103,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler, registerFunc func(datadog.
 	// Watch for changes to primary resource DatadogAgent
 	err = c.Watch(&source.Kind{Type: &datadoghqv1alpha1.DatadogAgent{}}, &handler.EnqueueRequestForObject{}, onCreate)
 	if err != nil {
-		return err
+	return err
```

Comment:
why?

Reply:
I'm not sure how that happened. Anyhow, I undid the change. Good spot!